docs/time/index.rst

```diff
@@ -1212,6 +1212,16 @@ jd         :class:`~astropy.time.TimeDeltaJD`
 datetime   :class:`~astropy.time.TimeDeltaDatetime`
 =========  ===================================================
 
+.. warning:: The ``datetime`` format does *not* support instants of time that
```

Comment:
The warning is for `TimeDelta` here -- it should instead be for regular `Time` - roughly line 218, maybe after the note about FITS.

Reply:
Also: datetime does not only not support instances that happen right at the leap seconds, it also makes all the related mistakes regarding actually elapsed time between instances.